Source/WebCore/Modules/WebGPU/GPUFeatureName.idl

```diff
@@ -35,5 +35,6 @@ enum GPUFeatureName {
     "texture-compression-etc2",
     "texture-compression-astc",
     "timestamp-query",
-    "indirect-first-instance"
+    "indirect-first-instance",
+    "bgra8unorm-storage"
```

Comment:
Can we keep these sorted?

Reply:
I think they should match: https://gpuweb.github.io/gpuweb/#enumdef-gpufeaturename

but that made me realize there are more missing, I will add those too